src/controller/editingcontroller.js

```diff
@@ -105,12 +105,12 @@ export default class EditingController {
 		}, { priority: 'low' } );
 
 		// Convert model markers changes.
-		this._listener.listenTo( this.model.markers, 'add', ( evt, name, range ) => {
-			this.modelToView.convertMarker( 'addMarker', name, range );
+		this._listener.listenTo( this.model.markers, 'add', ( evt, marker ) => {
+			this.modelToView.convertMarker( 'addMarker', marker.name, marker.getRange() );
```

Comment:
Because this function is not always used exactly with marker. This is more like saying "perform conversion of given part of data like it is in marker XXX". Take a look at `moveInOutOfMarker` and `insertMarkerOrIntoMarker` in `model-to-view-converters.js`.